tests/e2e/specs/in-game/one-offs/5_fives.spec.js

```diff
@@ -532,6 +532,8 @@ describe('FIVES', () => {
       cy.resolveFiveOpponent(Card.FIVE_OF_HEARTS);
       cy.get('#waiting-for-opponent-discard-scrim').should('not.exist');
 
+      cy.get('[data-opponent-hand-card=5-2]').should('be.visible');
```

Comment:
EXP: this verifies that the opponent's discarded card is revealed before the full state change